gui/App.cpp

```diff
@@ -0,0 +1,41 @@
+#include "App.h"
+
+App::App(SDL_Window *w, SDL_Renderer *r) {
```

Comment:
(Style) Do we add spaces between functions and parentheses? Will not note them in future, but we should choose one option and stick to it. I know that I added lots of spaces.